docs/blog/2023-09-06-distributed-key-generation.md

```diff
@@ -0,0 +1,154 @@
+
+Check this out:
+
+[GitHub - drand/drand: 🎲 A Distributed Randomness Beacon Daemon - Go implementation](https://github.com/drand/drand)
+
+[GitHub - drand/kyber: Advanced crypto library for the Go language](https://github.com/drand/kyber)
+
+### Sinking or Swimming: A Deep Dive into Pedersen's Security 🤿 :
+
+There has been considerable debate and, in some quarters, a level of confusion regarding the security aspects of Pedersen's DKG. To provide a clearer understanding of the evolving perspectives on its security, it's beneficial to delve into a brief timeline that outlines key milestones in the research focused on this framework.
+
+😃 🏊🏾‍♂️ ****1991: Pedersen**** 
+
+In the early 90s, Pedersen initially introduced the DKG scheme. The security in the initial scheme lies in the security of the Joint-Feldman Protocol for verifiable secret sharing during the secret-sharing process
+
+🙄⚓**********2006: [Gennaro et al.](https://link.springer.com/article/10.1007/s00145-006-0347-3)** 
+
+Gennaro et al. took the Joint-Feldman protocol as described by Pedersen and they showed that a potential adversary can influence the contribution of the public key in a way that is not being uniformly chosen randomly. Therefore, the adversary can take the advantage to compromise two of the participants and gain information about the private key, which means it leaks information about the secret. As a result, this is a sinking phase for Pedersen, as it shows that it cannot be used as a secure DKG protocol.
+
+😃 🏊🏾‍♂️ **2007**:  **[Rabin et al.](https://link.springer.com/content/pdf/10.1007/3-540-48910-X_21.pdf)** 
+
+In this 2007 study, Rabin et al. provided a series of security proofs showing that Feldman's (VSS) was susceptible to malicious inputs in Pedersen's (DKG). However, the same article also notes in Section 5 that Joint-Feldman (JF-DKG) has a beneficial attribute: it prevents any adversary from producing an output that allows the computation of its discrete logarithm. This feature is sufficiently robust to enable the construction of secure threshold signature schemes. Therefore, it makes Pedersen’s initial DKG sufficient for threshold signing. 
+
+😃 🏊🏾‍♂️ **************2021: [Gurkan et al](https://eprint.iacr.org/2021/005).**
+
+In a more recent study, Gurkan et al. showed that Pedersen's DKG is a secure way to handle rekeyable encryption schemes, signature schemes, and VUF schemes. Specifically, they provided proof that using JF-DKG is secure, preventing an adversary from forging a signature by participating in the actual DKG protocol.
+
+# ***The Future: Asynchronous DKG Algorithms***
+
+Looking ahead, one exciting development in the world of DKG is the adoption of asynchronous algorithms. Traditional DKG protocols often rely on synchronous communication assumptions, where participants have to wait for messages from others before proceeding.
```

Comment:
`participants have to wait for messages from others before proceeding.`
Not really, it's more they work in rounds, like they all have a perfect clock. They don't need to wait messages from others, but they do need to wait the end of each round to proceed to the next, regardless of the actions from others.
A protocol moving according to the messages received is exactly more in the spirit of async comm and is much easier to handle (not necessarily to devise and code though :D )